src/main/resources/lang/strings.json

```diff
@@ -228,12 +237,11 @@
     "voxelsniper.brush.sign-overwrite.info": "&3The arrow writes the internal line buffer to the target sign.\n&3The gunpowder reads the text of the target sign into the internal buffer.\n&3Colors can be used using the \"&\" symbol.\n&6Sign Overwrite Brush Parameters:\n&b/b sio 1 [set|toggle] (...) -- Sets the text of the first sign line. (e.g. /b sio 1 set Blah Blah | /b sio 1 toggle)\n&b/b sio 2 [set|toggle] (...) -- Sets the text of the second sign line. (e.g. /b sio 2 set Blah Blah | /b sio 2 toggle\n&b/b sio 3 [set|toggle] (...) -- Sets the text of the third sign line. (e.g. /b sio 3 set Blah Blah | /b sio 3 toggle\n&b/b sio 4 [set|toggle] (...) -- Sets the text of the fourth sign line. (e.g. /b sio 4 set Blah Blah | /b sio 4 toggle\n&b/b sio clear -- Clears the line buffer. (Alias: /b sio c)\n&b/b sio clearall -- Clears the line buffer and sets all lines back to enabled (Alias: /b sio ca)\n&b/b sio side [s] -- Sets the sign side to s. (not available in versions prior to 1.20)\n&b/b sio multiple [true|false] -- Enables or disables ranged mode. (see Wiki for more information) (Alias: /b sio m [true|false])\n&b/b sio save [n] -- Save you buffer to a file named n. (Alias: /b sio s [n])\n&b/b sio open [n] -- Loads a buffer from a file named n. (Alias: /b sio o [n])",
```

Comment:
Is this broken somehow? The formatting doesn't show up in green

Reply:
What is broken here :thinking: 
![image](https://github.com/IntellectualSites/fastasyncvoxelsniper/assets/43724816/a1060df6-627e-4e44-ad80-6ce824426df8)

Reply:
well that is weird, the highlighting was missing in the class but when it is shown up here right, we can ignore that